### PR TITLE
Server auth

### PR DESCRIPTION
**Pull request by jkp85**
Signed-off-by: Jan Kowalski <jankowalskipraca85@gmail.com>
Endpoint to validate server key.

### CODE REVIEW

**Comment by jkp85** [comment]
It should be sent as regular auth header `Authorization: Bearer token`. There are 2 steps of authenticating. First is regular api JWT authentication, second is checking if this key can be used to access server.

**Comment by johngriebel** [comment]
@jkp85 so what is the method of giving another user access to my server?

**Comment by jkp85** [comment]
To share api key. However now when I think about it you could just login with this key, which is not intended. I have to create custom verification view for regular tokens then, because there is no way currently to distinguish those tokens.

File: tbs_swagger.yml [hunk 1]
```diff
@@ -2731,6 +2731,33 @@ paths:
       summary: "Delete a server trigger"
       tags:
         - projects
+  '/v1/{namespace}/projects/{project_id}/servers/{id}/auth/':
```
Comment (jgwerner) [inline tbs_swagger.yml]: Lets call all ids by their type, such as `server_id` instead of `id`.

**Pull request merged**